daft/session.py

```diff
@@ -243,7 +243,7 @@ def create_table_if_not_exists(
 
         return catalog.create_table_if_not_exists(identifier, source, properties)
 
-    def create_temp_table(self, identifier: str, source: Schema | DataFrame) -> Table:
+    def create_temp_table(self, identifier: str, source: Schema | DataFrame) -> None:
```

Comment:
This is a breaking change, although I'm not super concerned about it since it was never actually returning a proper type that subclassed `Table`, so this is actually less misleading to users.